src/org/ods/orchestration/phases/DeployOdsComponent.groovy

```diff
@@ -48,6 +48,9 @@ class DeployOdsComponent {
             if (!openShiftDir.startsWith('openshift')) {
                 deploymentDescriptor.deployments.each { String deploymentName, Map deployment ->
                     importImages(deployment, deploymentName, project.sourceProject)
+                }
+                deploymentDescriptor.deployments.each { String deploymentName, Map deployment ->
+                    // importImages(deployment, deploymentName, project.sourceProject)
```

Comment:
Can you please remove the commented code?

Reply:
@jafarre-bi Done